python/foglamp/services/core/api/service.py

```diff
@@ -68,21 +68,18 @@ def get_service_records():
     Create a new service to run a specific plugin
 
     :Example:
-             curl -X POST /foglamp/service -d '{"name": "furnace4", "type": "south", "plugin": "dht11"}'
+             curl -X POST http://localhost:8081/foglamp/service -d '{"type": "south", "plugin": "dht11", "enabled": true}'
```

Comment:
No, this is completely wrong. The name is the important thing and the thing that defines a service, not the plugin. The same plugin maybe used by multiple services. Please remove this change.

Reply:
yes! `name` is required. See how `FogLAMP Storage` and `FogLAMP Core` got the names. the problem FOGL-1533 describes is, don;t consider the service name as plugin config to load!